alacritty/src/event.rs

```diff
@@ -314,9 +314,15 @@ impl<'a, N: Notify + 'a, T: EventListener> input::ActionContext<T> for ActionCon
             #[cfg(target_os = "freebsd")]
             let link_path = format!("/compat/linux/proc/{}/cwd", pid);
 
-            fs::read_link(link_path)
+            let mut args = fs::read_link(link_path)
                 .map(|path| vec!["--working-directory".into(), path])
-                .unwrap_or_default()
+                .unwrap_or_default();
+
+            // Adding currently loaded config-file in arguments.
+            args.push("--config-file".into());
+            args.extend(self.config.ui_config.config_paths.clone());
```

Comment:
We'd want to load all current arguments, not just config file. So we should lookup the current cmdline and just append it. That could be a bit of a problem if the working directory parameter already exists, but in general it is the most sensible solution.

Reply:
Agreed. Will append the arguments by reading it from command line arguments.
Also for windows the args variable expects it to be `Vec<String>` while on Unix its `Vec<OsStr>` is the difference expected?

Reply:
@chrisduerr any remarks?